Resources/Public/JavaScript/Test.js

```diff
@@ -0,0 +1,20 @@
+const menuConfig = {
```

Comment:
As we have real files in `Resources/Public/JavaScript/`, the `Resources/Public/JavaScript.gitkeep` is not needed anymore.

Reply:
yea it's removed already